msu/load.nut

```diff
@@ -7,6 +7,7 @@ local function includeLoad( _folder )
 
 includeLoad("ui");
 includeLoad("systems");
+includeLoad("hooks_helper");
```

Comment:
We discussed and we should probably expose this for when a mod adds another named item type?

Reply:
Yes that sounds good to me.